### PR TITLE
refactor: community picks alert messages

### PR DESCRIPTION
**Pull request by sshanzel**
### Describe what this PR does
- New Alert component with three different types as of the moment. 
- Utilize the alert component in Submit Article Modal based from the design

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-319 #done

### CODE REVIEW

File: packages/shared/src/components/modals/SubmitArticleModal.tsx [hunk 1]
```diff
@@ -132,33 +133,43 @@ export default function SubmitArticleModal({
...
+        <AlertParagraph>
+          Please do not add your own posts, promotional content, clickbait etc.
+          Additionally, redirected or shortened URLs will be rejected, please
+          submit the full and final URL.
+        </AlertParagraph>
+        <AlertParagraph>
+          For more details see our{' '}
+          <a
+            className="hover:underline text-theme-label-link"
+            href="https://docs.daily.dev/docs/how-does-daily-dev-work/community-picks-submission-guidelines"
```
Comment (rebelchris) [inline packages/shared/src/components/modals/SubmitArticleModal.tsx:166]: Just a sidenote, perhaps a nicer idea to put this in the const file with the other links.
Just makes it easier later on to re-use/change it in one spot.

File: packages/shared/src/components/modals/SubmitArticleModal.tsx [hunk 2]
```diff
@@ -132,33 +133,43 @@ export default function SubmitArticleModal({
...
+            target="_blank"
```
Comment (rebelchris) [inline packages/shared/src/components/modals/SubmitArticleModal.tsx:167]: Also probably add the rel noopener stuff just in case.

**Review by rebelchris — CHANGES_REQUESTED** [review]
Awesome, love the new component.
Some small remarks, but overall looks amazing

File: packages/shared/src/components/modals/SubmitArticleModal.tsx (hunk 2 above)
Comment (sshanzel) [reply packages/shared/src/components/modals/SubmitArticleModal.tsx:167]: Good point. Let me add it.

File: packages/shared/src/components/modals/SubmitArticleModal.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/modals/SubmitArticleModal.tsx:166]: Ah yes, totally agree with that! 🚢

**Review by rebelchris — APPROVED** [review]
Let's go 🚨

**Pull request merged**